src/composables/sdk13.ts

```diff
@@ -0,0 +1,176 @@
+const isNodeConnecting = ref<boolean>(false);
+const isNodeReady = ref<boolean>(false);
+const isNodeError = ref<boolean>(false);
+const aeppInfo: any = {};
+
+/**
+ * Composable that will replace the Vuex SDK plugin.
+ * For now, it works as an abstraction layer.
+ */
+export function useSdk13({ store }: IDefaultComposableOptions) {
+  const { isLoggedIn } = useAccounts({ store });
+  const isSdkReady = computed(() => !!sdk);
+  const activeNetwork = computed<INetwork>(() => store.getters.activeNetwork);
+
+  /**
+   * Create Node instance and get connection status
+   */
+  async function createNodeInstance(url: string) {
+    let nodeInstance;
+    isNodeReady.value = false;
+    isNodeError.value = false;
+    isNodeConnecting.value = true;
+    try {
+      // TODO: remove ignore version once HTTP compiler dependency is removed
+      nodeInstance = new Node(url, { ignoreVersion: true });
+      await nodeInstance.getStatus();
+      isNodeReady.value = true;
+    } catch (error) {
+      isNodeError.value = true;
+      return null;
+    } finally {
+      isNodeConnecting.value = false;
+    }
+    return nodeInstance;
+  }
+
+  async function initSdk() {
+    sdkBlocked = true;
+
+    await Promise.all([
+      watchUntilTruthy(() => store.state.isRestored),
+      watchUntilTruthy(isLoggedIn),
+    ]);
+    sdkCurrentNetwork = activeNetwork.value;
+    const nodeInstance = await createNodeInstance(activeNetwork.value.url);
+
+    sdk = new ShSdkWallet(store, {
+      name: 'Superhero',
+      nodes: [{
+        name: activeNetwork.value.name,
+        instance: nodeInstance!,
+      }],
+      id: 'Superhero Wallet',
+      type: WALLET_TYPE.extension,
+      onCompiler: new CompilerHttp(activeNetwork.value.compilerUrl),
+      onConnection(aeppId: string, params: any, origin: string) {
+        aeppInfo[aeppId] = { ...params, origin };
+      },
+      onDisconnect(__: any, client: any) {
+        client.disconnect();
+      },
+      async onSubscription(aeppId: string) {
+        const aepp = aeppInfo[aeppId];
+        const url = IS_EXTENSION_BACKGROUND ? new URL(aepp.origin) : new URL(origin);
+        const app = new App(url);
+        const { address } = store.getters.account;
+        if (
+          app instanceof App
+          && !(await store.dispatch('permissions/requestAddressForHost', {
+            host: app.host.host,
+            name: app.host.hostname,
+            address,
+            connectionPopupCb: async () => (IS_EXTENSION_BACKGROUND
+              ? showPopup(app.host.href, POPUP_TYPE_CONNECT)
+              : store.dispatch('modals/open', {
+                name: MODAL_CONFIRM_CONNECT,
+                app: {
+                  name: app.host.hostname,
+                  icons: [],
+                  protocol: app.host.protocol,
+                  host: app.host.host,
+                },
+              })),
+          }))
+        ) return Promise.reject(new RpcRejectedByUserError('Rejected by user'));
```

Comment:
I'd suggest adding braces here as the return may blend in with if's conditions
```suggestion
        ) {
          return Promise.reject(new RpcRejectedByUserError('Rejected by user'));
        }
```